### ISSUE TITLE
HTTP Client - 403 Access Denied

### ISSUE THREAD

**Issue by lvaldovinos** [substack/stream-adventure#122]
Hi,

While i was finishing on this problem, i realized Spotify's using 8099 port, so if you are listening to music on Spotify while solving this problem, test No.2 will fail with 403 code, stating "Access is denied".

Solving this is easy, you just got to make sure nobody is using 8099 port before solving this problem.

Maybe instead of listening to 8099 you may change the port to another one that is not likely to be used by other applications.

Regards.

**Comment by calvinmetcalf** [comment]
yeah we can pick a new port, how about 27391 (chosen with [this](https://www.random.org/integers/?num=1&min=1&max=40000&col=5&base=10&format=html&rnd=new))

**Comment by ma-santamaria** [comment]
Hello,

I also have encountered this problem (spotify + programming is quite common)

I know that find a free port is difficult, but how about a notice in [problem.txt](https://github.com/substack/stream-adventure/blob/master/problems/http_client/problem.txt)?

Regards.

**Comment by mjjc** [comment]
+1. You could lookup a free port using [node-portfinder](https://github.com/indexzero/node-portfinder) and pass it to a solution through command-line arguments.

**Comment by rafaeleyng** [comment]
Same problem here.
Solved by closing Spotify while submitting the exercise.
